src/views/dashboard/index.vue

```diff
@@ -166,10 +166,12 @@ export default {
       this.axios.get(this.services['statutory'] + '/mine').then((statutoryResponse) => {
         const input = eventsResponse.data.data.concat(statutoryResponse.data.data)
         for (const event of input) {
-          if (moment().isSameOrBefore(event.starts)) {
-            this.events.future.push(event)
-          } else {
-            this.events.past.push(event)
+          if (!event.applications[0].cancelled && event.applications[0].status != 'rejected') {
```

Comment:
I tried a few things, but could never get more than 1 application in this list. Is that possible, and thus will there be a problem when doing this?

Reply:
You are only checking the first application (applications[0]), so this will be a problem

Reply:
You can loop through all the received applications, but probably there is also a nicer way. Maybe somebody on Slack has ideas?

Reply:
Well you are checking the first of your applications to the event, and I don't think you can ever have more than one application from yourself to an event

Reply:
Oh wait, you are getting event objects back of course. Then my comment is invalid

Reply:
Through the API you can create multiple applications for one event if I see the function behind the endpoint. You should be able to test that locally through Postman (I think)